telegram/ext/filters.py

```diff
@@ -958,30 +994,42 @@ def filter(self, message: Message) -> bool:
 class _Dice(MessageFilter):
     __slots__ = ("emoji", "values")
 
-    def __init__(self, values: Optional[SCT[int]] = None, emoji: Optional[DiceEmojiEnum] = None):
+    def __init__(
+        self,
+        values: Optional[SCT[int]] = None,
+        emoji: Optional[DiceEmojiEnum] = None,
+    ):
         super().__init__()
         self.emoji: Optional[DiceEmojiEnum] = emoji
-        self.values: Optional[Collection[int]] = [values] if isinstance(values, int) else values
+        self.values: Optional[Collection[int]] = (
+            [values] if isinstance(values, int) else values
+        )
 
         if emoji:  # for filters.Dice.BASKETBALL
             self.name = f"filters.Dice.{emoji.name}"
-            if self.values and emoji:  # for filters.Dice.Dice(4)  SLOT_MACHINE -> SlotMachine
+            if (
+                self.values and emoji
+            ):  # for filters.Dice.Dice(4)  SLOT_MACHINE -> SlotMachine
```

Comment:
can you revert such formatting changes? They are not required and are cluttering this PR.

Reply:
Let me take a look on those changes then, thanks for the review